packages/frontend/src/components/SettingsValidator/ValidatorTable/index.tsx

```diff
@@ -44,6 +51,119 @@ const Icon = ({ validationError }: { validationError: ValidationResponse }) => {
+) => {
+    if ('chartViews' in validationError) return validationError.chartViews;
+    if ('dashboardViews' in validationError)
+        return validationError.dashboardViews;
+};
+
+const TableValidationItem = forwardRef<
+    HTMLTableRowElement,
+    {
+        projectUuid: string;
+        validationError: ValidationResponse;
+    }
+>(({ projectUuid, validationError }, ref) => {
+    const { mutate: deleteValidation } = useDeleteValidation(projectUuid);
+    const theme = useMantineTheme();
+
+    const { hovered, ref: isHoveredRef } = useHover<HTMLTableRowElement>();
+
+    return (
+        <tr
+            ref={mergeRefs(ref, isHoveredRef)}
+            onClick={() =>
+                handleOnValidationErrorClick(projectUuid, validationError)
+            }
+        >
+            <td>
+                <Anchor
+                    sx={{
+                        color: 'unset',
+                        ':hover': {
+                            color: 'unset',
+                            textDecoration: 'none',
+                        },
+                    }}
+                    href={getLinkToResource(validationError, projectUuid)}
+                    target="_blank"
+                >
+                    <Flex gap="sm" align="center">
+                        <Icon validationError={validationError} />
+
+                        <Stack spacing={4}>
+                            <Text fw={600}>
+                                {getErrorName(validationError)}
+                            </Text>
+
+                            {(isChartValidationError(validationError) ||
+                                isDashboardValidationError(
+                                    validationError,
+                                )) && (
+                                <Text fz={11} color="gray.6">
+                                    {getViews(validationError)} view
+                                    {getViews(validationError) === 1 ? '' : 's'}
+                                    {' • '}
+                                    {validationError.lastUpdatedBy ? (
+                                        <>
+                                            Last edited by{' '}
+                                            <Text span fw={500}>
+                                                {validationError.lastUpdatedBy}
+                                            </Text>
+                                        </>
+                                    ) : null}
+                                </Text>
+                            )}
+                        </Stack>
+                    </Flex>
+                </Anchor>
+            </td>
+            <td>
+                <ErrorMessage validationError={validationError} />
+            </td>
+            <td>
+                <Tooltip label="Dismiss error" position="top">
+                    <Box w={24} h={24}>
+                        {hovered && (
+                            <IconCircleX
```

Comment:
all tabler icons thare are used using `MantineIcon` will get a benefit of passing mantine variables which will not require using a mantine theme hook

`<MantineIcon icon={IconCircleX} color='gray.6' />`

Reply:
also, good alternative to IconCircleX IconCircleXFilled (circle-x-filled) and can be little smaller `<MantineIcon icon={IconCircleX} color='gray.6' size='md' />`